Formula/c/cargo-sweep.rb

```diff
@@ -0,0 +1,18 @@
+class CargoSweep < Formula
+  desc "Utility for cleaning up unused build files generated by Cargo"
+  homepage "https://github.com/holmgr/cargo-sweep"
+  url "https://github.com/holmgr/cargo-sweep/archive/refs/tags/v0.7.0.tar.gz"
+  sha256 "21004272ffacbe19b5e2b6c521a021eb55abd24e96599fc5049c650abb708af2"
+  license "MIT"
+  head "https://github.com/holmgr/cargo-sweep.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--no-default-features", *std_cargo_args
+  end
+
+  test do
+    assert_equal "cargo-sweep #{version}", shell_output(bin/"cargo-sweep -V").strip
```

Comment:
can you check out the other plugin and write a more substantial test with `rust-init`?

Reply:
I was thinking about it. However, it is not so  clear what would be a good test. The utility deletes unused Cargo build files. For that: 1) cargo has to be used to compile something. 2) some days have to pass 3) then the build resources will be deleted.

To model this, one would need to set the time-stamps of some of the build files. I am afraid however that such a procedure will be very hard to maintain as the build files of cargo as well as the setup of the build directories might change in the future.